Hospital.Domain/Enums/BloodGroup.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Hospital.Domain.Enums;
+
+/// <summary>
+/// Represents the blood group of a patient.
+/// </summary>
+public enum BloodGroup
+{
+    /// <summary>
+    /// Blood group O (I).
+    /// </summary>
+    O,
+
+    /// <summary>
+    /// Blood group A (II).
+    /// </summary>
+    A,
+
+    /// <summary>
+    /// Blood group B (III).
+    /// </summary>
+    B,
+
+    /// <summary>
+    /// Blood group AB (IV).
+    /// </summary>
+    AB
```

Comment:
Кажется я такие замечания уже писал. Необходимо акронимы именовать также в PascalCase.
```suggestion
    Ab
```